salt/metalk8s/defaults.yaml

```diff
@@ -113,7 +113,6 @@ kube_api:
   cert:
     server_signing_policy: kube_apiserver_server_policy
     client_signing_policy: kube_apiserver_client_policy
```

Comment:
Could be squashed with the previous commit, no?

Reply:
Could be;
one commit => add module + add module path to salt_tree.
one commit => use module in sls files while replacing variables were need be.
one commit => remove the hard coded service IP addresses within the `defaults.yaml`.

Looks ok to me

Reply:
@Ebaneck I agree with Sylvain, one commit to introduce functionality (the module), one to use the functionality (replace values where need be AND remove now-unused values). If you keep the last commit as you suggest, its value is roughly the same as a `fix typo` commit, which we don't like much ;)